lib/Driver/ToolChains.h

```diff
@@ -43,15 +43,48 @@ class LLVM_LIBRARY_VISIBILITY GenericUnix : public ToolChain {
   InvocationInfo constructInvocation(const AutolinkExtractJobAction &job,
                                      const JobContext &context) const override;
 
+  /// If provided, and if the user has not already explicitly specified a
+  /// linker to use via the "-fuse-ld=" option, this linker will be passed to
+  /// the compiler invocation via "-fuse-ld=". Return an empty string to not
+  /// specify any specific linker (the "-fuse-ld=" option will not be
+  /// specified).
+  ///
+  /// The default behavior is to use the gold linker on ARM architectures,
+  /// and to not provide a specific linker otherwise.
   virtual std::string getDefaultLinker() const;
 
+  /// Whether to specify a linker -rpath to the Swift runtime library path.
+  /// -rpath is not supported on all platforms, and subclasses may override
+  /// this method to return false on platforms that don't support it. The
+  /// default is to return true (and so specify an -rpath).
   virtual bool shouldProvideRPathToLinker() const;
 
+  /// Whether to explicitly specify the target triple as the the linker
```

Comment:
Thanks for pointing this out! Fixed in https://github.com/apple/swift/commit/bb20005b01508f426c6365769f27770bea2a9996 (and https://github.com/apple/swift/commit/c0e1a7ce3f7498c2be97b1d8c8c6f15d6891b640, sorry for the additional typo!!).